openhands/server/session/session_config.py

```diff
@@ -0,0 +1,18 @@
+
+
+from dataclasses import dataclass
+
+
+@dataclass
+class SessionConfig:
```

Comment:
Yes I think so, and once it goes there... it should probably do what the others do (or it will become more clear where the differences need to be)

It's not obvious to me why do we duplicate attributes like llm_model as opposed to giving this class an llm_config with a particular user-chosen model, but could be wrong

Reply:
Ah I see, it's not really a config in the sense we normally use that word--it can't be globally set. Let's rename this to something like `SessionInitData`